redirects.txt

```diff
@@ -138,7 +138,10 @@ inventory/settings/warehouses.rst inventory/management/warehouses.rst
 inventory/settings/warehouses/difference_warehouse_location.rst inventory/management/warehouses/difference_warehouse_location.rst   # settings/* --> management/* (#424)
 inventory/settings/warehouses/location_creation.rst inventory/management/warehouses/location_creation.rst 							# settings/* --> management/* (#424)
 inventory/management/delivery/scheduled_dates.rst inventory/management/planning/scheduled_dates.rst 								# delivery/scheduled_dates --> planning/scheduled_dates (#660)
-inventory/management/planning/scheduled_date.rst inventory/manamgent/planning/scheduled_dates.rst 									# scheduled_date --> scheduled_dates (#660)
```

Comment:
was this already merged? 😕

Reply:
Yes, the reviewer of that PR and I apparently didn't see that mistake ^^'